roles/kubernetes-apps/ansible/tasks/coredns.yml

```diff
@@ -1,4 +1,14 @@
 ---
+- name: Kubernetes Apps | Set CoreDNS Proxy for resolvconf_mode = 'host_resolvconf'
+  set_fact:
+    coredns_proxy: "{{ upstream_dns_servers|join(' ') }}"
+  when:
+    - dns_mode in ['coredns', 'coredns_dual']
+    - inventory_hostname == groups['kube-master'][0]
+    - resolvconf_mode == 'host_resolvconf'
```

Comment:
Why do we set it only for host_resolvconf mode? Let's just set it for both cases